python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -937,11 +935,8 @@ def _create_table_optimize_block(self, pserver_index, pserver_program,
             "LearningRate": [lr_var]
         }
         outputs = {"ParamOut": [param_var]}
-        table_opt_block.append_op(
-            type=table_opt_op.type,
-            inputs=inputs,
-            outputs=outputs,
-            attrs=table_opt_op.attrs)
+        # only support sgd now
+        table_opt_block.append_op(type="sgd", inputs=inputs, outputs=outputs)
```

Comment:
Could print a warning here to indicate users.

Reply:
done